website/content/en/docs/tasks/operator.md

```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
 - **tag:** This field determines the version of project images to pull. By default, it uses the `latest` version.
 
 All the flags are mapped to corresponding field in the ClusterLink custom resource.
+
+## Manual Deployment without CLI
+
+To deploy the ClusterLink project without using the CLI, follow the instructions below:
+
+1. Download the configuration files (CRDs, operator RBACs, and deployment) from GitHub:
+
```

Comment:
Can one get these from the release installation files instead of cloning the entire repo from git? Or at least from specific BLOB paths on GitHub?

Instructions should be directed to a user, not a developer.

Reply:
I didn't find an easy way to do that (just cloning the config folder https://github.com/clusterlink-net/clusterlink/tree/main/config)